src/Controller/User/UserMapViewController.java

```diff
@@ -314,6 +362,15 @@ private void initialize() throws Exception {
 
         panel.setVisible(false);
         directionPaneView();
+
+        for( Building b : model.getHospital().getBuildings()) {
+            if(b.getFloors().contains(model.getCurrentFloor())) {
+                for(Floor f : b.getFloors()) {
+                    dropDown.getItems().add(Integer.toString(f.getFloorNumber()));
```

Comment:
Will be super weird for multiple buildings.... Consider asking building names to the string or something?